### PR TITLE
Make channel checking to pass (optionally) #4

### PR DESCRIPTION
**Pull request by thomashopkins32**
Added -i or --ignore-exception flag to continue execution after listing forbidden channel packages.

### CODE REVIEW

File: nsls2forge_utils/check_results.py [hunk 1]
```diff
@@ -21,6 +22,9 @@ def check_conda_channels(forbidden_channel='conda-forge',
         environment
     cmd: str, optional
         a command to check a list of packages in a conda environment
+    ignore_exception: bool, optional
+        a command to print the list of packages from the channels which are forbidden
+        and proceed without exiting
```
Comment (mrakitin) [inline nsls2forge_utils/check_results.py:27]:
```suggestion
        and proceed without exiting if set to True
```

File: nsls2forge_utils/check_results.py [hunk 2]
```diff
@@ -21,6 +22,9 @@ def check_conda_channels(forbidden_channel='conda-forge',
         environment
     cmd: str, optional
         a command to check a list of packages in a conda environment
+    ignore_exception: bool, optional
+        a command to print the list of packages from the channels which are forbidden
```
Comment (mrakitin) [inline nsls2forge_utils/check_results.py:26]:
```suggestion
        a flag to print the list of packages from the channels which are forbidden
```

File: nsls2forge_utils/check_results.py [hunk 3]
```diff
@@ -41,8 +45,12 @@ def check_conda_channels(forbidden_channel='conda-forge',
 
     if failed_packages:
         formatted = '\n'.join(failed_packages)
-        raise RuntimeError(f'Packages from the "{forbidden_channel}" channel '
-                           f'found:\n{formatted}')
+        if ignore_exception:
+            print(f'Packages from the "{forbidden_channel}" channel '
+                  f'found:\n{formatted}')
```
Comment (mrakitin) [inline nsls2forge_utils/check_results.py:50]: I think the contents being printed in either case can be pulled above to a var like `msg`, and be reused here. DRY (don't repeat yourself) principle will be satisfied :)

File: nsls2forge_utils/cli.py [hunk 4]
```diff
@@ -27,6 +27,13 @@ def check_results():
                         help=('a command to check a list of packages in a '
                               'conda environment'))
 
+    # Ignore forbidden channel exception to continue execution
+    parser.add_argument('-i', '--ignore-exception', dest='ignore_exception',
+                        action='store_true',
+                        help=('a command to print the list of packages from '
```
Comment (mrakitin) [inline nsls2forge_utils/cli.py:33]: This should be sync'd with the corresponding function's docstring.

**Comment by thomashopkins32** [comment]
Yes, I will work on that today. Thank you.

**Pull request merged**